packages/api/test/pin.spec.js

```diff
@@ -78,7 +78,7 @@ const createPinRequest = async (cid, token) => {
   })).json()
 }
 
-describe('Pinning APIs endpoints', () => {
+describe.only('Pinning APIs endpoints', () => {
```

Comment:
Please remove

Reply:
😓
Sorry about that, so annoying.
I wonder if we should consider adding a step in the CI to catch these things.